spec/approvals/parser/basic

```diff
@@ -0,0 +1 @@
+["svg", {"a"=>"b"}, [["rect", {"width"=>"30"}, []]]]
```

Comment:
Pretty minor but looks like git is complaining about a missing newline? Adding it might cause issues with the spec.

Reply:
My git is not complaining, do you have some hooks to that effect? But yes, these approvals should not be altered manually.

Reply:
I don't have any hooks setup, I think it is standard git behaviour to complain about files that don't have a newline at the end. I think you would see a message when doing a `git diff` or a `git add -p`.

Github shows a red arrow when there is no newline at the end of the file.

Don't think it is that important though so feel free to ignore this.